src/WebApp/Startup.cs

```diff
@@ -39,6 +44,15 @@ public void ConfigureServices(IServiceCollection services)
                     options.LowercaseUrls = true;
                 });
 
+            services.Configure<MailOptions>(_configuration.GetSection("Mail"));
+
+            services.Configure<MailOptions>(options =>
+            {
+                options.Sender = "Codidact";
+            });
```

Comment:
This is not needed. I added that in the example to demonstrate that it is possible to do this.